### DISCUSSION TITLE
Run multiple Appium Server and session for Appium 2.0

### DISCUSSION THREAD

**Discussion by justATester3939** [appium/appium#18441] [Ideas]
I am using Robot framework with Python.
Here is my situation, I have only one macbook to code and run daily automation test case. 
Please make easy way to run multiple Appium Server and session for example:
+ 1 server for run automation on IOS
+ 1 server for run automation on Android
+ 1 server for develop new automation script

Please help!

**Comment by saikrishna321** [comment]
@justATester3939 You can run single server session and run all the tests. Appium server can handle multiple client sessions
